sdk/include/aditof/camera.h

```diff
@@ -148,6 +148,20 @@ class SDK_API Camera {
     virtual Status requestFrame(Frame *frame,
                                 FrameUpdateCallback cb = nullptr) = 0;
 
+    /**
+     * @brief Scale AB image with logarithmic base 10.
+     * @param frame - The frame of the camera
+     * @param normalizeSteps - If is 0 then the algorithm is not applied,
+     * if is 1 is applied the normalization between 0 and 255,
+     * if is 2 is applied normalization between 0 and 255 and log10
+     * if is another value the algorithm is not applied
+     * @param[out] abData - The new data is returned throught this pointer
```

Comment:
This doesn't seem to be needed. Client code can always get it though:
frame->getData("ab", abData)